validator/project_validator.go

```diff
@@ -662,7 +662,21 @@ func validateBVFields(project *model.Project) ValidationErrors {
 					break
 				}
 			}
-			if !taskHasValidDistro {
+			if !taskHasValidDistro && task.IsGroup {
```

Comment:
If we re-factor the logic from 658 to 689 to a lil helper function, we could just return true on taskHasValidDistro instead of setting to true and casing on it each time

Reply:
done

Reply:
don't need to check taskHasValidDistro here or in the else since we break for it above